src/Microsoft.Identity.Client/Platforms/iOS/TokenCacheAccessor.cs

```diff
@@ -206,7 +217,12 @@ private void RemoveAll(string service)
             {
                 foreach (var record in records)
                 {
-                    SecKeyChain.Remove(record);
+                    var result = SecKeyChain.Remove(record);
+
+                    if (result != SecStatusCode.Success)
+                    {
+                        _requestContext.Logger.Warning("Failed to remove cache record from iOS Keychain: SecStatusCode." + result.ToString());
```

Comment:
Remember to reference the issue that this is addressing. I think it's #546 - which is already closed, but reference it anyway